http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SamAuthProvider.scala

```diff
@@ -181,10 +182,12 @@ class SamAuthProvider[F[_]: Effect: Logger: Timer: OpenTelemetryMetrics](samDao:
     creatorEmail: WorkbenchEmail,
     googleProject: GoogleProject
   )(implicit sr: SamResource[R], encoder: Encoder[R], ev: Ask[F, TraceId]): F[Unit] =
-    if (sr.policyNames == Set(SamPolicyName.Creator))
+    // TODO: consider using v2 for all existing entities if this works out for apps https://broadworkbench.atlassian.net/browse/IA-2569
```

Comment:
I feel like `createResource` vs `createResourceV2` is confusing because we're using Sam v2 APIs in both cases.

How about `createResource` and `createResourceWithParent`?

You can keep the if statement, just want to make naming more clear.